Sources/SwiftParsec/CharacterSet.swift

```diff
@@ -0,0 +1,86 @@
+//==============================================================================
+// CharacterSet.swift
+// SwiftParsec
+//
+// CharacterSet compatibility wrapper
+//==============================================================================
+
+import struct Foundation.CharacterSet
+
+#if _runtime(_ObjC)
+
+//==============================================================================
```

Comment:
Could you remove the separator comment (//===...) and add a '/' to the slashes of the comment so that it can be included in the documentation.